tests/python/test_element_wise.py

```diff
@@ -12,7 +13,7 @@ def rand(dtype):
     if ti.core.is_integral(dtype):
         return randint(1, 5)
     else:
-        return float(randint(1, 5)) / 5
+        return float(randint(1, 5)) / 5 - 0.01  # prevent floordiv step
```

Comment:
Not sure what this means...sorry.

Reply:
Because GLSL doesn't provide `pow(int, int)`, it only provide `pow(float, float)`, so:
```py
pow(4.999, 3) = 124
pow(5.001, 3) = 125
```
And due to binary floating-point problems, `5` becames `4.99`, therefore:
```py
pow(5, 4) = 124
```

Reply:
Sounds good. (Btw should the comment be something like "prevent integer operands in pow in GLSL")?

Reply:
Because it's the same case with floordiv... let's say `prevent integer operands in pow and floordiv in GLSL` then.